src/platform/tools_api.py

```diff
@@ -232,13 +232,19 @@ def run_tool(
     if "clusterId" not in data.keys():
         data["clusterId"] = get_default_cluster_id(client=client, org_key=org_key)
 
-    if "approveAmount" not in data.keys():
-        data["approveAmount"] = 0
```

Comment:
changed behavior -- now we don't pass any approveAmount -- that defaults to the org default